### ISSUE TITLE
Hero Video: Bad heading contrast

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#68]
On some of the video scenes, the white heading and paragraph text does not have sufficient contrast of 4.5:1 with the background. 		

This will be specific from video to video, but may be solved by a solid background behind the text.
